index.js

```diff
@@ -93,6 +96,12 @@ module.exports = {
       }
     }
     var self
+
+
+    ssb.since((value) => {
+      p.push(value)
```

Comment:
pull-pushable uses a buffer until there are consumers of the stream. So if it turns out that `sinceStream` is not used anywhere, pull-pushable will still buffer all the log in an array in memory! :warning:

I think we need more careful consideration on this implementation. Perhaps it should be a lazy source, that only begins reading the log once the source is pulled for the first time.

Reply:
No good! I just switched over to [pull-notify](https://github.com/pull-stream/pull-notify/), which seems to resolve this by **creating one stream per consumer** and **only pushing to active streams**. Does that seem more reasonable to you?